src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -80,6 +80,13 @@ trait Blockchain {
   def getMptNodeByHash(hash: ByteString): Option[MptNode]
 
 
+  def getAccount(address: Address): Option[Account] = {
+    val hash = crypto.kec256(address.bytes)
+    getMptNodeByHash(hash).collect {
```

Comment:
Not exactly, `getMptNodeByHash` fetches a MPT node by its hash from the storage, in this case we would need to construct a new MPT from the state root hash to get the corresponding leaf.